packages/components/pie-card-container/src/index.ts

```diff
@@ -58,12 +68,52 @@ export class PieCardContainer extends LitElement implements CardContainerProps {
                 target=${this.target || nothing}
                 rel=${this.rel || nothing}
                 aria-label=${this.aria?.label || nothing}
-                aria-disabled=${this.disabled ? 'true' : 'false'}>
+                aria-disabled=${this.disabled ? 'true' : 'false'}
+                style=${paddingCSS || nothing}>
                     <slot></slot>
                 </div>
             </a>`;
     }
 
+    /**
+     * Generates padding for the component based on `padding` values passed
+     * by the consumer.
+     *
+     *
+     * Example: 'a' or 'a, b'
+     * Single values i.e `'a'` applies to all sides and `'a, b'` applies to: top & bottom, left & right
+     *
+     * @private
+     */
+    private generatePaddingCSS (): string {
+        const { padding } = this;
+        let paddingCSS = '';
+
+        // Check if padding is empty (null, undefined, or an empty string)
+        if (!padding) {
+            return '';
+        }
+
+        const paddingArray = padding
+            .split(',')
+            .map((item) => item.trim())
+            .filter((value) => /^[a-g]$/.test(value));
```

Comment:
Does `a, z` result in just a value of `a`?

Reply:
Yup that's correct.